Code/Tools/SceneAPI/SceneBuilder/Importers/AssImpBoneImporter.cpp

```diff
@@ -71,6 +71,13 @@ namespace AZ
                 FindAllBones(scene, boneByNameMap);
 
                 bool isBone = FindFirstBoneByNodeName(currentNode, boneByNameMap);
+
+                // Export the scene root node as a bone.
+                if (!isBone)
+                {
+                    isBone = (currentNode == scene->mRootNode);
```

Comment:
uber nit: The parens here are redundant and could be removed

Reply:
I had habit from previous jobs that has strict coding standard to always add the parens in case like this. Main reason is for code readability.

Reply:
the not condition where currentNode != scene->mRootNode is covered where !isBone

- isBone: True
- isBone: false
  - c == b: True
  - c != b : false  

```
isBone |=  (currentNode == scene->mRootNode)
```

or

```
bool isBone = FindFirstBoneByNodeName(currentNode, boneByNameMap) || currentNode == scene->mRootNode;
```